os/arch/arm/src/s5j/s5j_rtc_lowerhalf.c

```diff
@@ -290,29 +291,29 @@ static struct s5j_lowerhalf_s g_rtc_lowerhalf = {
  * Public Functions
  ****************************************************************************/
 /****************************************************************************
- * Name: s5j_rtc_lowerhalf
+ * Name: s5j_rtc_initialize
  *
  * Description:
- *   Instantiate the RTC lower half driver for the S5J.
- *   General usage:
- *
- *     #include <tinyara/rtc.h>
- *     #include "s5j_rtc.h"
- *
- *     struct rtc_lowerhalf_s *lower;
- *     lower = s5j_rtc_lowerhalf();
- *     rtc_initialize(0, lower);
+ *   Register the RTC driver to provide RTC functionality
```

Comment:
In my opinion, lower half driver of RTC is correct, instead of RTC driver, function name too.